opentelemetry/metadata.csv

```diff
@@ -0,0 +1,38 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Some of these metrics are not actually sent right now but are featured in the OpenTelemetry specification and I added them in case they are sent in the future. For reference, the issue tracking this is open-telemetry/opentelemetry-collector-contrib#2220